engine-tests/src/main/java/org/terasology/engine/Environment.java

```diff
@@ -128,7 +128,7 @@ protected void setupBlockManager(AssetManager assetManager) {
         // empty
     }
 
-    protected void setupExtraDataManager(Context context) {
+    protected void setupExtraDataManager(Context ctxt) {
```

Comment:
This is another instance of HiddenFieldCheck where I don't like either of the options:

- add a bloated `@SuppressWarnings(...)` annotation
- rename the parameter to `ctxt` or `newContext` or similar, actually making it worse to read 

@keturn how would you resolve these warnings (I think you somewhat defended that check rule before)? Unfortunately, configuration options are limited...